roles/contiv/tasks/ovs.yml

```diff
@@ -0,0 +1,24 @@
+---
+- include: packageManagerInstall.yml
+  when: source_type == "packageManager"
+  tags:
+    - binary-update
+
+- name: OVS | Configure selinux for ovs
+  shell: semanage permissive -a openvswitch_t
+
+- name: OVS | Enable ovs
+  service: name=openvswitch enabled=yes
+
+- name: OVS | Start ovs
+  service: name=openvswitch state=started
+  register: ovs_started
+
+- name: OVS | Configure ovs
+  shell: "ovs-vsctl set-manager {{ item }}"
+  with_items:
+    - "tcp:127.0.0.1:6640"
+    - "ptcp:6640"
```

Comment:
Does this need to be persisted in some way or does ovs-vsctl persist it in a config file somewhere?